src/app/components/header-mobile/header-mobile.component.html

```diff
@@ -1,61 +1,59 @@
-    <nav class="navbar">
-      <div class="navbar-container">
-
-        <div class="burger-button-container">
-            <button (click)="toggle()" class="burger-button">
-              @if(isMobileSideNavOpen){
-                <img src="assets/pictures/close.svg" class="large-icon" img> 
-              }
-              @else{
-                <img src="assets/pictures/burger-menu.svg" class="large-icon" img> 
-              }
-                 
-            </button>        
-          </div>
-
-
-        <div class="logo">
-          <a routerLink="home"><img class="logo-image" src="assets/pictures/Blop.png" alt="Logo"></a>
-        </div>
-
-       
-        <div class="connection-container">
-          <button (click)="openDialog()"><img class="connection-logo" src="assets/pictures/Connexion.png" alt="Connection"></button>
-        </div>
-  
-  
+<header class="header" [ngClass]="{ 'blur-background': isMobileSideNavOpen }">
+  <nav class="navbar">
+    <div class="navbar-container">
+      <div class="burger-button-container">
+        <button (click)="toggle()" class="burger-button">
+          @if(isMobileSideNavOpen){
+          <img src="assets/pictures/close.svg" class="large-icon" img />
+          } @else{
+          <img src="assets/pictures/burger-menu.svg" class="large-icon" img />
+          }
+        </button>
       </div>
-    </nav>
-  </header>
-
-  @if(isMobileSideNavOpen){
-    <div class="sidenav">
-        <ul>
-            <a routerLink="home"><li>Création De Personnage</li></a>
-            <li>
-              Jeu
-              <ul class="submenu">
-                <a routerLink="home"><li>Introduction</li></a>
-                <a routerLink="home"><li>Règles</li></a>
-              </ul>
-            </li>
-            <li>
-              Glossaire
-              <ul class="submenu">
-                <a routerLink="home"><li>Races</li></a>
-                <a routerLink="glossaire/classes"><li>Classes</li></a>
-                <a routerLink="home"><li>Les Secrets</li></a>
-                <a routerLink="home"><li>Croyances</li></a>
-                <a routerLink="glossaire/lieux"><li>Lieux</li></a>
-              </ul>
-            </li>
-  
-            <a routerLink="home"><li routerLink="home">News</li></a>
-            <a routerLink="home"><li routerLink="home">Studio</li></a>
-          </ul>
 
+      <div class="logo">
+        <a routerLink="home"
+          ><img class="logo-image" src="assets/pictures/Blop.png" alt="Logo"
+        /></a>
       </div>
-  }
-
 
+      <div class="connection-container">
+        <button (click)="openDialog()">
+          <img
+            class="connection-logo"
+            src="assets/pictures/Connexion.png"
+            alt="Connection"
+          />
+        </button>
+      </div>
+    </div>
+  </nav>
+</header>
+
+@if(isMobileSideNavOpen){
+<div class="sidenav">
+  <ul>
+    <a routerLink="home"><li>Création De Personnage</li></a>
```

Comment:
you'll probably have conflicts for this display. If not, a tags should be inside li tags and not the opposite.

Do It on Both header components